aws/cloud-formation/batchSystem/batchSystem.yaml

```diff
@@ -203,17 +203,17 @@ Resources:
 
   BatchSNSTopic:
     Type: AWS::SNS::Topic
-    Properties: 
+    Properties:
       DisplayName: "AWS Batch Job Has Failed"
       FifoTopic: FALSE
-      Subscription: 
+      Subscription:
```

Comment:
Outoja whitespace muutoksia ilmaanutunut vaikka IntelliJ ei git committia tehdessä näitä näyttänyt diff